<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1250</width>
      <height>650</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1250</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>650</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>1250</width>
       <height>650</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1250</width>
       <height>650</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1250</width>
       <height>650</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(224, 221, 220, 212);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>368</x>
        <y>126</y>
        <width>870</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: rgb(255, 255, 255);
background-color: rgb(255, 227, 83);*/

QWidget{
background-color: #bbddff;;
color:#black;
}

QHeaderView::section{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(30, 100, 170, 255), stop:1 rgba(5, 75, 255, 255));
padding:2px;
border:1px solid #ffffff;
font-size:14pt;
}

QTableWidget{
gridline-color:#99ff99;
font-size:12pt;
border:1px solid #000000;

} 

QTableWidget QTableCornerButton::section{
background-color: #0000ff;
border:1px solid #ff0000;
}
QScrollBar::handle:vertical
{
background-color: #ff0000;
    min-height: 28px;
    border-radius: 2px;
}
QScrollBar::handle:horizontal
{
background-color: #ff0000;
    min-width: 5px;
    border-radius: 2px;
}

</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>categorie</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>question</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>juste</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>faux1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>faux2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>faux3</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>127</y>
        <width>361</width>
        <height>517</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #bbddff;;
border:1px solid #000000;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_57">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>السؤال</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_58">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>180</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>صحيح</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_59">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>280</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>خاطيء 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_60">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>380</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>خاطيء 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_61">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>480</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>خاطيء 3</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_question">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPlainTextEdit
{

background-color: rgb(255, 255, 255);
border:0px;
border-bottom:2px solid gray;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="locale">
        <locale language="Arabic" country="Egypt"/>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_juste">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border:0px;
border-bottom:2px solid gray;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="locale">
        <locale language="Arabic" country="Egypt"/>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_faux1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border:0px;
border-bottom:2px solid gray;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_faux2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border:0px;
border-bottom:2px solid gray;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_faux3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border:0px;
border-bottom:2px solid gray;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="locale">
        <locale language="Arabic" country="Egypt"/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>369</x>
        <y>556</y>
        <width>868</width>
        <height>88</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #bbddff;;
border:1px solid #000000;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="comboCat_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>25</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(90, 90, 0, 255), stop:1 rgba(155, 155, 255, 255));
font: 20pt &quot;Traditional Arabic&quot;;</string>
       </property>
       <property name="currentText">
        <string notr="true">الجزائر</string>
       </property>
       <item>
        <property name="text">
         <string>الجزائر</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ابتدائي</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ثانية-ف1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ثانية-ف2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ثانية-ف3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>رابعة-ف1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>رابعة-ف2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>رابعة-ف3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>العالم</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>85</x>
         <y>62</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>النوع</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboTyp_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>25</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Traditional Arabic</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(90, 90, 0, 255), stop:1 rgba(155, 155, 255, 255));
font: 20pt &quot;Traditional Arabic&quot;;
font: 20pt &quot;Traditional Arabic&quot;;
</string>
       </property>
       <property name="locale">
        <locale language="Arabic" country="Algeria"/>
       </property>
       <property name="currentText">
        <string>تاريخ</string>
       </property>
       <item>
        <property name="text">
         <string>تاريخ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>جغرافية</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btn_insert_3">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>50</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">   QPushButton{   
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 120, 190, 255), stop:1 rgba(55, 55, 255, 255));
font: 20pt &quot;Traditional Arabic&quot;;    
        color: white;
font: 20pt &quot;Traditional Arabic&quot;;
border: 1px solid '#ffffff';
        border-radius: 7px;

}
           QPushButton:hover{
            background: '#00ff00';
            color: white;
            font: 20pt &quot;Traditional Arabic&quot;;
            border: 1px solid '#ffffff';
            border-radius: 7px;
        }
</string>
       </property>
       <property name="text">
        <string>تسجيل</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_update_3">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>25</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">   QPushButton{   
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 120, 190, 255), stop:1 rgba(55, 55, 255, 255));
font: 20pt &quot;Traditional Arabic&quot;;    
        color: white;
font: 20pt &quot;Traditional Arabic&quot;;
border: 1px solid '#ffffff';
        border-radius: 7px;
}
           QPushButton:hover{
            background: '#00ff00';
            color: white;
            font: 20pt &quot;Traditional Arabic&quot;;
            border: 1px solid '#ffffff';
            border-radius: 7px;
        }

</string>
       </property>
       <property name="text">
        <string>تحديث</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>62</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
border:1px solid blue;</string>
       </property>
       <property name="text">
        <string>الفئة</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_delete_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>50</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">   QPushButton{   
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 120, 190, 255), stop:1 rgba(55, 55, 255, 255));
font: 20pt &quot;Traditional Arabic&quot;;    
        color: white;
font: 20pt &quot;Traditional Arabic&quot;;
border: 1px solid '#ffffff';
        border-radius: 7px;

}
           QPushButton:hover{
            background: '#00ff00';
            color: white;
            font: 20pt &quot;Traditional Arabic&quot;;
            border: 1px solid '#ffffff';
            border-radius: 7px;
        }</string>
       </property>
       <property name="text">
        <string>حذف</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_new_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>25</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">   QPushButton{   
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 120, 190, 255), stop:1 rgba(55, 55, 255, 255));
font: 20pt &quot;Traditional Arabic&quot;;    
        color: white;
font: 20pt &quot;Traditional Arabic&quot;;
border: 1px solid '#ffffff';
        border-radius: 7px;

}
           QPushButton:hover{
            background: '#00ff00';
            color: white;
            font: 20pt &quot;Traditional Arabic&quot;;
            border: 1px solid '#ffffff';
            border-radius: 7px;
        }</string>
       </property>
       <property name="text">
        <string>جديد</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>10</y>
         <width>411</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>311</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>411</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="top_right_btns">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>10</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="minimizeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">   QPushButton{   
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 120, 190, 255), stop:1 rgba(55, 55, 255, 255));    
        color: white;
        font-family: 'shanti';        
	    font: 75 30pt &quot;MS Shell Dlg 2&quot;;
        border-radius: 12px;
padding-top:-5px;

}
           QPushButton:hover{
            background: '#00ff55';
            color: red;
            font: 75 30pt &quot;MS Shell Dlg 2&quot;;
border-radius: 12px;
        }</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">

   QPushButton{   
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 120, 190, 255), stop:1 rgba(55, 55, 255, 255));
        color: white;
        font-family: 'shanti';        
	    font: 75 13pt &quot;MS Shell Dlg 2&quot;;
        border-radius: 12px;
padding-top:-2px;
}
           QPushButton:hover{
            background: '#ff4444';
            color: white;
            font: 75 13pt &quot;MS Shell Dlg 2&quot;;
border-radius: 12px;
        }</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
